model_analyzer/result/result_manager.py

```diff
@@ -202,7 +202,7 @@ def create_tables(self,
                     for gpu_metric in gpu_metrics:
                         if gpu_metric.tag not in gpu_specific_metrics_from_measurements:
                             gpu_specific_metrics_from_measurements[
-                                gpu_metric.tag] = gpu_metric
+                                gpu_metric[0]] = gpu_metric
```

Comment:
I don't understand why this change was needed (shouldn't you still need the .tag?)

Reply:
good catch. I confirmed that [0] is incorrect. May have been a leftover from a bad merge between some of your changes?